.travis.yml

```diff
@@ -26,10 +26,7 @@ before_install:
 - php --version
 - composer install
 script:
-- npm run force-build
-- npm run test-base
-- npm run test-base-ws
-- pip list
+- ./build.sh 2>&1
```

Comment:
imho, we could merge `build.sh` and `tests-manager.sh` in one `.sh` file, which can be controlled easier (i.e. providing the needed array of exchanges that need re-test)

at this moment, you need to duplicate code from build.sh into tests-manager

Reply:
Yes I also considered it, but first we must make sure Igor/Carlo are ok with the tests change (test only modified exchange every time instead of the 24h rule)

Reply:
> Yes I also considered it, but first we must make sure Igor/Carlo are ok with the tests change (test only modified exchange every time instead of the 24h rule)

have you asked them already and waiting for an answer? or I should tag them here?

Reply:
@ttodua I think I will probably change first and ask later. After the c# merge our build time will increase a lot, so I also believe we need to find a way of improve it

Reply:
@ttodua can you check it now?

Reply:
give me a moment

Reply:
looks good to me.
maybe now delete tests-manager.sh file and remove that commented line too.

Reply:
@ttodua I would leave the tests-manager script for now in the repo, it contains an important implementation that can be reused in the future